### ISSUE TITLE
(Optimize) You don't need this 1 here

### ISSUE THREAD

**Issue by CharlesDLandau** [BogdanPro/intro-data-capstone-musclehub#4]
We want to avoid clutter and my solution doesn't have that. I think it's cleaner to remove the ', 1' in there.

https://github.com/BogdanPro/intro-data-capstone-musclehub/blob/932c534db396a1cde11eb10c88ac4152165c9777/musclehub_project_bogdan_protsenko/musclehub.py#L172

**Comment by BogdanPro** [comment]
Yeah, for sure it is more clear without redundant assignments. Can I reupload my solution? Will you have some time to review it along with other changes? @CharlesDLandau
